skiko/src/commonMain/kotlin/org/jetbrains/skia/DirectContext.kt

```diff
@@ -5,8 +5,20 @@ import org.jetbrains.skia.impl.Library.Companion.staticLoad
 import org.jetbrains.skiko.RenderException
 import org.jetbrains.skiko.loadOpenGLLibrary
 
-class DirectContext internal constructor(ptr: NativePointer) : RefCnt(ptr) {
+class DirectContext : RefCnt {
+
+    internal constructor(ptr: NativePointer) : super(ptr) {
+         DirectContext_nSetResourceCacheLimit(ptr, resourceCacheLimit)
+    }
+
     companion object {
+
+        /**
+         *  GPU resource cache limit in bytes used by DirectContext instances.
+         *  Set the property on the early startup of an application to take effect.
+         */
+        var resourceCacheLimit: Long = 256 * 1024 * 1024 // the same as Skia's default, see GrResourceCache.h, kDefaultMaxSize
```

Comment:
- Skiko serves as a implementation details for Compose, so if we want to have an API for Compose users, we have to expose it there, and then delegate it Skiko
- making it a global variable is not very good from API perspective. It is better to make it local, and then set it via chain of calls (for example, from ComposeWindow -> SkiaLayer -> RedrawerFactory)
- also, we have to make it local because Skiko by design should have the same API as Skia, and in Skia it is local, not a global property.

Reply:
> read this global property inside every Redrawer we have.

It also better to pass it via `SkiaLayerProperties`, every redrawer has access to it